bugsnag-android-core/src/main/java/com/bugsnag/android/RootDetector.kt

```diff
@@ -69,6 +78,22 @@ internal class RootDetector(
         return false
     }
 
+    /**
+     * Checks the contents of /system/build.prop to see whether it contains dangerous properties.
+     * These properties give a good indication that a phone might be using a custom
+     * ROM and is therefore rooted.
+     */
+    fun checkBuildProps(): Boolean {
+        runCatching {
+            return buildProps.bufferedReader().useLines { lines ->
+                lines.filter { line ->
+                    line.contains("ro.debuggable=[1]") || line.contains("ro.secure=[0]")
```

Comment:
How do we rate the risk of a false positive here?  E.g. what if the line was commented out or just the end of a completely unrelated property?  Both are arguably unlikely, but I wonder if we should tighten it up a little?

Reply:
Tweaked detection so that it trims whitespace and checks that the start of the line begins with these properties (which discounts the case where the line is commented out)